src/libraries/Common/src/System/Net/Http/Http3/QPack/H3StaticTable.cs

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+
+#if KESTREL
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3.QPack
+#else
+namespace System.Net.Http.QPack
+#endif
+{
+    internal class H3StaticTable
+    {
+        private static readonly H3StaticTable _instance = new H3StaticTable();
+
+        private readonly Dictionary<int, int> _statusIndex = new Dictionary<int, int>
+        {
+            [103] = 24,
+            [200] = 25,
+            [304] = 26,
+            [404] = 27,
+            [503] = 28,
+            [100] = 63,
+            [204] = 64,
+            [206] = 65,
+            [302] = 66,
+            [400] = 67,
+            [403] = 68,
+            [421] = 69,
+            [425] = 70,
+            [500] = 71,
+        };
+
+        private readonly Dictionary<HttpMethod, int> _methodIndex = new Dictionary<HttpMethod, int>
+        {
+            // TODO connect is internal to system.net.http
```

Comment:
I'll stop commenting on these, but it'd be good to include issue links for all the TODOs.